semanticdb/semanticdb3/guide.md

```diff
@@ -0,0 +1,576 @@
+      caches their converted counterparts in a configurable directory.
+      This only works for jars - class directories are never cached.
+    </td>
+    <td>
+      Platform-specific user-specific cache directory whose name includes
+      <code>semanticdb</code> and version number of Metacp.
+    </td>
+  </tr>
+  <tr>
+    <td>
+      <code>--exclude-scala-library-synthetics</code>,<br/>
+      <code>--include-scala-library-synthetics</code>
+    </td>
+    <td></td>
+    <td>
+      Specifies whether the output classpath should include a jar that contains
+      SemanticDB files corresponding to definitions missing from
+      <code>scala-library.jar</code>, e.g. <code>scala.Any</code>,
+      <code>scala.AnyRef</code> and others.
+    </td>
+    <td>
+      <code>--exclude-scala-library-synthetics</code>
+    </td>
+  </tr>
+  <tr>
+    <td><code>--par</code>,<br/><code>--no-par</code></td>
+    <td></td>
+    <td>
+      Toggles parallel processing. If enabled, classpath entries
+      will be converted in parallel according to the strategy chosen
+      by parallel collections of the Scala standard library.
+      <br/><br/>
+      NOTE: Some of our users <a href="https://github.com/scalameta/scalameta/issues/1398">have reported deadlocks</a> supposedly caused
+      by enabling <code>--par</code>. Proceed at your own risk.
+    </td>
+    <td>
+      <code>--no-par</code>
+    </td>
+  </tr>
+</table>
+
+## Consuming SemanticDB
+
+### Scalameta
+
+Scalameta includes the `semanticdb3` library that contains
+[ScalaPB](https://scalapb.github.io/) bindings to
+[the SemanticDB protobuf schema](semanticdb3.proto). Using this library,
+one can model SemanticDB entities as Scala case classes and serialize/deserialize
+them into bytes and streams. For more information, check out [the Scaladoc](https://static.javadoc.io/org.scalameta/semanticdb3_2.11/3.7.4/index.html#scala.meta.internal.semanticdb3.TextDocuments).
+
+NOTE: At this point, it's not very easy to take a classpath, traverse it and
+load all available SemanticDB payloads. SemanticDB-based tools are currently
+responsible for implementing discovery and deserialization on their own.
+However, we recognize this as a usability problem and are planning to
+[improve the situation](https://github.com/scalameta/scalameta/issues/1566)
+very soon.
+
+For examples of developer tools that consume SemanticDB, take a look at:
+  * [Scalafix](https://github.com/scalacenter/scalafix/)
+  * [Metadoc](https://github.com/scalameta/metadoc)
+  * [Metals](https://github.com/scalameta/metals)
+
+### Protoc
+
+The Protocol Compiler tool (`protoc`) can inspect protobuf payloads in
+`--decode` (takes a schema) and `--decode_raw` (doesn't need a schema) modes.
+For the reference, here's [the SemanticDB protobuf schema](semanticdb3.proto).
+
+```
+$ tree
+.
+├── META-INF
+│   └── semanticdb
+│       └── Test.scala.semanticdb
+└── Test.scala
+
+$ protoc --proto_path <directory with the .proto file>\
```

Comment:
bravo 👏 you figured out the mystical incantation to get this working 😂 

I probably spent 10-20 minutes trying to figure out until I gave up. I have the following alias in my machine however
```
alias semanticdb="protoc --decode_raw"
```
and use it like this
```
cat foo.semanticdb | semanticdb